File: lab2/task3.cpp
#include <iostream>

using namespace std;

int main() {
    int n = 0;

    while (n < 3 || n > 9) {
        cout << "please enter a number between 3 and 9: ";
        cin >> n;
        if (n < 3 || n > 9) {
            cout << "invalid input." << endl;
        }
    }

    for (int i = 1; i <= 2 * n - 1; i++) {
        int a = n - i;

        if (a < 0) {
            a = -a;
        }

        int k = n - a;


        for (int j = 0; j < k; j++) {
            cout <<"*";
        }

        cout << endl;
    }

    return 0;
}

File: lab2/task3.py
# Task3
n = 0
while n < 3 or n > 9:
    n = int(input("Please enter a number between 3 and 9: "))
    if n < 3 or n > 9:
        print("Invalid number")

for i in range(1, 2 * n):

    ece = n - i
    if ece < 0:
        ece = -ece

    m = n - ece
    print("*" * m)
